### DISCUSSION TITLE
setting and mapping

### DISCUSSION THREAD

**Discussion by nsupegemini** [toluaina/pgsync#379] [Q&A]
Hi @toluaina,

Thank you for such an awesome package. 

I am trying to add `setting` and `mapping` based on [book](https://github.com/toluaina/pgsync/blob/main/examples/book/schema.json) example, however it is not creating analyzer. Am I missing something?

**Comment by toluaina** [comment]
Hi, thanks and sorry for the delay. are you directly using the book example?
My first guess is the version of Elasticsearch/Opensearch you are using.
Can you provide more details on version number?

**Reply by nsupegemini** [reply]
Thank you for getting back. It was mistake on my side. It is working now. Sorry I forgot to update here
